### PR TITLE
[DPM-95] add ws auth method & fix repos

### PR DESCRIPTION
**Pull request by algys**

### CODE REVIEW

File: server/session/manager.go [hunk 1]
```diff
@@ -2,53 +2,88 @@ package session
...
+	"github.com/google/uuid"
 	"github.com/gorilla/websocket"
 	"github.com/rs/zerolog/log"
+	"platform-backend/models"
 	"platform-backend/server/api"
 	"sync"
 )
 
-type Manager struct {
+type ManagerImpl struct {
```
Comment (SCaptainCAP) [inline server/session/manager.go:14]: Maybe need to make this a repository? And create "localstorage repository" for session manager as for other models+usecases

File: server/app.go [hunk 2]
```diff
@@ -22,11 +22,17 @@ import (
...
+type RefreshRequest struct {
+	RefreshToken string `json:"refreshToken"`
+}
+
 type App struct {
 	httpServer     *http.Server
 	config         *config.Config
 	wsUpgrader     websocket.Upgrader
-	sessionManager *session.Manager
+	sessionManager session.Manager
```
[the remark this answers is not in the document]
Comment (algys) [reply server/app.go:35]: Manager is interface)

File: auth/usecase/usecase.go [hunk 3]
```diff
@@ -4,71 +4,219 @@ import (
...
-	token := jwt.New(jwt.SigningMethodHS256)
-	token.Claims = jwt.MapClaims{"account_name": user.AccountName}
-
-	signed, err := token.SignedString(a.jwtSecret)
-	if err != nil {
-		log.Debug().Msgf("JWT token sign error, %s", err.Error())
-		return "", err
-	}
-
+func (a *AuthUseCase) SignUp(ctx context.Context, user *models.User) (string, string, error) {
```
[the remark this answers is not in the document]
Comment (algys) [reply auth/usecase/usecase.go:34]: added

File: go.mod [hunk 4]
```diff
@@ -3,16 +3,13 @@ module platform-backend
...
 
 require (
-	github.com/auth0/go-jwt-middleware v0.0.0-20190805220309-36081240882b
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/golang-migrate/migrate/v4 v4.10.0
+	github.com/google/uuid v1.1.1
 	github.com/gorilla/websocket v1.4.2
-	github.com/jackc/pgconn v1.4.0
 	github.com/jackc/pgx/v4 v4.5.0
 	github.com/randallmlough/pgxscan v0.0.0-20191120191928-b26f55b9d3f4
```
[the remark this answers is not in the document]
Comment (algys) [reply go.mod]: removed

File: server/session/manager.go (hunk 1 above)
Comment (algys) [reply server/session/manager.go]: moved

**Pull request merged**